python/setup.py

```diff
@@ -103,6 +103,8 @@
         "opentelemetry-exporter-otlp==1.1.0"
     ]
 }
+if sys.version_info >= (3, 7, 0):
+    extras["k8s"].append("kopf")
```

Comment:
this may not work, but let's see :)

Reply:
I think it's actually not important here -- just installing in the image is enough.

Reply:
it actually broke support for py36 ref #26886, as no versions of `kopf>=1` support py36 (yet ray is still committed to py36, couldnt find a tracker or deprecation timeline btw)

Reply:
The issue is with documentation, which is missing this fact: py36 is not supported in the operator image, but it is perfectly fine to use a py37 operator image with py36 Ray images as long as the Ray versions are the same.